_includes/v21.2/sidebar-data/deploy.json

```diff
@@ -0,0 +1,416 @@
+                  ]
+                },
+                {
+                  "title": "Operate CockroachDB on Kubernetes",
+                  "items": [
+                    {
+                      "title": "Pod Scheduling",
+                      "urls": [
+                        "/${VERSION}/schedule-cockroachdb-kubernetes.html"
+                      ]
+                    },
+                    {
+                      "title": "Resource Management",
+                      "urls": [
+                        "/${VERSION}/configure-cockroachdb-kubernetes.html"
+                      ]
+                    },
+                    {
+                      "title": "Certificate Management",
+                      "urls": [
+                        "/${VERSION}/secure-cockroachdb-kubernetes.html"
+                      ]
+                    },
+                    {
+                      "title": "Cluster Scaling",
+                      "urls": [
+                        "/${VERSION}/scale-cockroachdb-kubernetes.html"
+                      ]
+                    },
+                    {
+                      "title": "Cluster Monitoring",
+                      "urls": [
+                        "/${VERSION}/monitor-cockroachdb-kubernetes.html"
+                      ]
+                    },
+                    {
+                      "title": "Cluster Upgrades",
+                      "urls": [
+                        "/${VERSION}/upgrade-cockroachdb-kubernetes.html"
+                      ]
+                    },
+                    {
+                      "title": "Optimizing Performance",
+                      "urls": [
+                        "/${VERSION}/kubernetes-performance.html"
+                      ]
+                    }
+                  ]
+                },
+                {
+                  "title": "OpenShift Deployment",
+                  "urls": [
+                    "/${VERSION}/deploy-cockroachdb-with-kubernetes-openshift.html"
+                  ]
+                },
+                {
+                  "title": "Multi-Cluster Deployment",
+                  "urls": [
+                    "/${VERSION}/orchestrate-cockroachdb-with-kubernetes-multi-cluster.html"
+                  ]
+                }
+              ]
+            },
+            {
+              "title": "Manual Deployment",
+              "items": [
+                {
+                  "title": "Overview",
+                  "urls": [
+                    "/${VERSION}/manual-deployment.html"
+                  ]
+                },
+                {
+                  "title": "On-Premises",
+                  "urls": [
+                    "/${VERSION}/deploy-cockroachdb-on-premises.html",
+                    "/${VERSION}/deploy-cockroachdb-on-premises-insecure.html"
+                  ]
+                },
+                {
+                  "title": "AWS",
```

Comment:
I think all these cloud options should include the word Deployment.

Reply:
Added "Deploy on...."